doc/LETS_ENCRYPT_CONFIG.md

```diff
@@ -0,0 +1,49 @@
+Here are the steps to get Let's Encrypt SSL certificates on Chef server
+
+##Prerequisites
+
+1. Chef server must be reachable on port 80 or 443 on a public IP address
```

Comment:
Yes, as long as Let's encrypt can validate with the self signed certificate that is installed out-of-the-box. Let's encrypt will try TLS first, then HTTP.
I am currently testing this scenario